app/mailers/user_notifications.rb

```diff
@@ -188,11 +193,12 @@ def notification_email(user, opts)
     use_site_subject = opts[:use_site_subject]
     add_re_to_subject = opts[:add_re_to_subject]
     show_category_in_subject = opts[:show_category_in_subject]
+    original_username = @notification.data_hash[:original_username] || @notification.data_hash[:display_username]
```

Comment:
Because in case of "Invite user to a Message", [we are passing](https://github.com/discourse/discourse/blob/master/app/models/topic.rb#L549) `display_username` to notifications data. So with this current code, username is not getting shown in "Invited to PM" notification emails.

Another way to handle this case would be passing `original_username` as well to notifications data.